### PR TITLE
Us36 posts likes

### PR DESCRIPTION
**Pull request by jervalles**

### CODE REVIEW

**Review by Gumatan — DISMISSED** [review]

File: backend/index.js [hunk 1]
```diff
@@ -99,6 +112,45 @@ app.post("/api/posts", (req, res) => {
   });
 });
 
+app.put(
+  "/api/posts/:id/like",
+  passport.authenticate("jwt", { session: false }),
+  (req, res) => {
+    const postId = parseInt(req.params.id);
+    const formData = req.body;
+    console.log(req.user);
```
Comment (LoicBrassart) [inline backend/index.js]: Nope

File: backend/index.js [hunk 2]
```diff
@@ -99,6 +112,45 @@ app.post("/api/posts", (req, res) => {
...
+
+    const userId = req.user.id;
+    const userLike = formData.userLike;
+    if (userLike)
+      db.query(
+        "INSERT INTO `like` (user_id,post_id) VALUES (?,?)",
+        [userId, postId],
+        (err, results) => {
+          if (err) {
+            console.log(err);
```
Comment (LoicBrassart) [inline backend/index.js]: Nope

File: backend/index.js [hunk 3]
```diff
@@ -99,6 +112,45 @@ app.post("/api/posts", (req, res) => {
...
+          }
+        }
+      );
+    else
+      db.query(
+        "DELETE FROM `like` WHERE user_id=? AND post_id=?",
+        [userId, postId],
+        (err, results) => {
+          if (err) {
+            console.log(err);
```
Comment (LoicBrassart) [inline backend/index.js]: Nope

File: frontend/src/components/NewsFeed.jsx [hunk 4]
```diff
@@ -28,6 +28,7 @@ function NewsFeed() {
     if (offsetPosts === 0) {
       axios.get(`${backend}/api/posts/${offsetPosts}`).then(({ data }) => {
         setPosts(data);
+        console.log(data);
```
Comment (LoicBrassart) [inline frontend/src/components/NewsFeed.jsx]: nope

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Nope.

**Review by Gumatan — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
No More Evil

**Pull request merged**